src/commands/aas/common.ts

```diff
@@ -44,6 +45,10 @@ export abstract class AasCommand extends Command {
     description: 'Where you write your logs. For example, /home/LogFiles/*.log or /home/LogFiles/myapp/*.log',
   })
 
+  private isDotnet = Option.Boolean('--dotnet', false, {
+    description: 'Add in required .NET-specific configuration options, is automatically inferred for code runtimes',
```

Comment:
Would add something about this being a good option for container based customers